src/functionalTest/resources/application-functional.yml

```diff
@@ -1,4 +1,4 @@
-targetInstance: ${TEST_URL:http://localhost:8090}
+targetInstance: ${TEST_URL:https://rd-professional-api-pr-574.service.core-compute-preview.internal}
```

Comment:
Any reason you changed from `http://localhost:8090` ?

Reply:
by mistake committed and will revert